src/Core/AdminConsole/Services/EventLoggingListenerService.cs

```diff
@@ -37,7 +37,7 @@ internal async Task ProcessReceivedMessageAsync(string body, string messageId)
             }
             else
             {
-                _logger.LogError($"An error occurred while processing message {messageId} - Invalid JSON");
+                _logger.LogError("An error occured while processing message: {MessageId} - Invalid JSON", messageId);
```

Comment:
```suggestion
                _logger.LogError("An error occurred while processing message: {MessageId} - Invalid JSON", messageId);
```

Did Sonar suggest this? Two Rs!